File: apps/api/app/schemas/intelligence.py
from datetime import datetime
from pydantic import BaseModel


class IntelligenceEventRead(BaseModel):
    id: str
    headline: str
    source_url: str | None = None
    timestamp: datetime
    credibility_score: float
    relevance_score: float
    flagged_reason: str | None = None

File: packages/shared/types/intelligence.ts
export type IntelligenceEvent = {
  id: string;
  headline: string;
  body?: string;
  sourceUrl?: string;
  timestamp: string;
  credibilityScore: number;
  relevanceScore: number;
  flaggedReason?: string;
  duplicateReports?: IntelligenceEvent[];
};

